src/iotjs_module_process.cpp

```diff
@@ -130,7 +130,7 @@ static JResult WrapEval(const char* source) {
   int len2 = strlen(source);
   int len3 = strlen(wrapper[1]);
 
-  String code("", len1 + len2 + len3 + 1);
+  String code("", len1 + len2 + len3);
```

Comment:
Sorry, I don't want to stole your credit. :) Of course I'll wait #330 to land first. This patch is currently a WIP patch, not ready for merge, but may help someone who want to use the latest version of JerryScript.